server/mockedData.js

```diff
@@ -0,0 +1,632 @@
+exports.items = [
```

Comment:
put array into variable (e.g. `const USERS = [...]`) - then work with variable and export it if needed:

```js
module.exports = { items: USERS };
```